.github/workflows/build-lint-test.yml

```diff
@@ -30,36 +30,25 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: 22
-      - name: Install dependencies
+      - name: Install npm dependencies
         run: npm ci
         working-directory: ./frontend
-      - name: Build
+      - name: Build frontend
         run: npm run build
         working-directory: ./frontend
       # Compile and test backend
+      - name: Install musl
+        run: sudo apt-get update && sudo apt-get install -y musl-tools
       - name: Setup Rust
         shell: bash
         run: >
           rustup update stable &&
           rustup default stable &&
-          echo RUST_VERSION=$(rustc --version | cut -d' ' -f2) >> $GITHUB_ENV
+          rustup target add x86_64-unknown-linux-musl
       - name: Cargo cache
-        uses: actions/cache@v4
+        uses: Swatinem/rust-cache@v2
         with:
-          path: |
-            ~/.cargo/.crates.toml
-            ~/.cargo/.crates2.json
-            ~/.cargo/bin/
-            ~/.cargo/registry/index/
-            ~/.cargo/registry/cache/
-            ~/.cargo/git/db/
-            ./backend/target/
-          key: ${{ runner.os }}-cargo-${{ env.RUST_VERSION }}-${{ hashFiles('**/Cargo.lock') }}
-          restore-keys: ${{ runner.os }}-cargo-${{ env.RUST_VERSION }}-
-      - name: Install musl
```

Comment:
We have to keep `sudo apt-get update && sudo apt-get install -y musl-tools` in order to compile for musl.

Reply:
It might not the installed on the runner: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md

Reply:
See also https://github.com/actions/runner-images/issues/5770#issuecomment-1160218740

Reply:
It's still there, just moved up by a few lines